.vscode/launch.json

```diff
@@ -8,11 +8,14 @@
             "name": "Listen for Xdebug (remote/docker)",
             "type": "php",
             "request": "launch",
-            "port": 9000,
+            "hostname": "0.0.0.0",
```

Comment:
Is it necessary to specify the hostname?

Reply:
Yes, it doesn't connect back if we don't.